libraries/localizeAppleDeviceModel/v1/code.js

```diff
@@ -0,0 +1,246 @@
+// == iPhone ==
+const iPhone = {
+    "iPhone1,1": "iPhone 1",
+    "iPhone1,2": "iPhone 3G",
+    "iPhone2,1": "iPhone 3GS",
+    "iPhone3,1": "iPhone 4",
+    "iPhone3,2": "iPhone 4",
+    "iPhone3,3": "iPhone 4",
+    "iPhone4,1": "iPhone 4S",
+    "iPhone5,1": "iPhone 5",
+    "iPhone5,2": "iPhone 5",
+    "iPhone5,3": "iPhone 5c",
+    "iPhone5,4": "iPhone 5c",
+    "iPhone6,1": "iPhone 5s",
+    "iPhone6,2": "iPhone 5s",
+    "iPhone7,1": "iPhone 6 Plus",
+    "iPhone7,2": "iPhone 6",
+    "iPhone8,1": "iPhone 6s",
+    "iPhone8,2": "iPhone 6s Plus",
+    "iPhone9,1": "iPhone 7",
+    "iPhone9,2": "iPhone 7 Plus",
+    "iPhone9,3": "iPhone 7",
+    "iPhone9,4": "iPhone 7 Plus",
+    "iPhone10,1": "iPhone 8",
+    "iPhone10,4": "iPhone 8",
+    "iPhone10,2": "iPhone 8 Plus",
+    "iPhone10,5": "iPhone 8 Plus",
+    "iPhone10,3": "iPhone X",
+    "iPhone10,6": "iPhone X",
+    "iPhone11,2": "iPhone XS",
+    "iPhone11,6": "iPhone XS Max",
```

Comment:
I found also an "iPhone11,4": "iPhone XS Max".

Reply:
Would you like to cross check?

Reply:
I've checked it and found some references where `iPhone11,4` and `iPhone11,6` were tagged as iPhone XS Max. Adding `iPhone11,4`:`iPhone XS Max` to the mappings as well.